adoc/Service-Broker-Setup.adoc

```diff
@@ -15,11 +15,9 @@ ifdef::env-github[]
 endif::[]
 // End attribute entry list
 
-
```

Comment:
I would rather not remove the line breaks before the sections. They give a bit more structure when editing the document and viewing the sources.